src/helpers/error-message.helper.ts

```diff
@@ -145,9 +145,7 @@ export class ErrorMessageHelper {
         title = `${error.worker}`;
         break;
       case ParseErrorCode.UNKNOWN_VALUE:
-        message = `Nieznana wartość zmiany: "${error.actual}" w dniu ${
-          error.day! + 1
-        }. Przyjęto, że zmiana to wolne.`;
+        message = `Nieznana wartość zmiany: "${error.actual}" w dniu ${error.day! + 1}.`;
         type = ScheduleErrorType.ILLEGAL_SHIFT_VALUE;
         title = `${error.worker}`;
```

Comment:
Tytuły się nie wyświetlają, a akurat w tym przypadku warto byłoby pokazać tego pracownika, żeby dokładniej naprowadzić na miejsce błędu.
![titles](https://user-images.githubusercontent.com/48785655/111267893-e1ce2a80-862c-11eb-9d86-b34ad4f6124e.png)

Reply:
Bardzo słuszna uwaga!
Ale myślę, że to jest poza skopem tego zadania, bo tutaj już potrzeba będzie zmieniać pracowników.
Myślę, że będzie lepiej jeśli zrobimy to w tym osobnym zadaniu z ujednoliceniem komunikatów błędów.